docs/DEBUG.md

```diff
@@ -1 +1,18 @@
 # Debugging Guide  
+## Electron Process  
+TODO: create a vscode launch.json configuration that attaches to the elctron process. https://stackoverflow.com/a/41073851   
+
+## Renderer Process  
+1. Start app: `npm run start`.  
+2. Open Chrome Dev Tools: `F12` in browser, `Ctrl + Shift + I` in electron.  
+3. Open `Sources` tab.  
+4. `Ctrl + Shift + F` and search for the file/function/code you would like to break into.  
+5. Place breakpoint.  
+6. Trigger breakpoint.  
+> Note: Reloading the page may be required if the code executes at startup.  
+
+## Unit Tests  
+1. Download & install `Jest` VSCode plugin.  
+2. Open the test's source file.  
+3. A `Debug` button should appear above every test. Place breakpoints, and click debug.  
```

Comment:
Debug button appears only if the test fails.  Anytime I've to debug a test I need first to let it fails and sometime Jest get lost.  Is there a config option to always have jest add the debug button even for successful tests ?

Reply:
Found how to always debug tests in VS studio:

- Set a breakpoint in a test file or files.
- Go to the Debug view, select the 'Jest All' configuration, then press F5 or click the green play button.
- Your breakpoint will now be hit.

source: https://github.com/Microsoft/vscode-recipes/tree/master/debugging-jest-tests